e2e-testing/docker-compose-e2e-test.yml

```diff
@@ -0,0 +1,229 @@
+version: '3'
+services:
+  kafka:
+    image: blacktop/kafka:2.6
+    ports:
+      - 9092:9092
+    environment:
+      KAFKA_ADVERTISED_HOST_NAME: kafka
+      KAFKA_CREATE_TOPICS:
+        "to-ender:1:1,\
+        to-vulcan:1:1,\
+        to-websockets-orderbooks:1:1,\
+        to-websockets-subaccounts:1:1,\
+        to-websockets-trades:1:1,\
+        to-websockets-markets:1:1,\
+        to-websockets-candles:1:1"
+      KAFKA_LISTENERS: INTERNAL://:9092,EXTERNAL_SAME_HOST://:29092
+      KAFKA_ADVERTISED_LISTENERS: INTERNAL://kafka:9092,EXTERNAL_SAME_HOST://localhost:29092
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: INTERNAL:PLAINTEXT,EXTERNAL_SAME_HOST:PLAINTEXT
+      KAFKA_INTER_BROKER_LISTENER_NAME: INTERNAL
+      DD_AGENT_HOST: datadog-agent
+    healthcheck:
+      test: ["CMD-SHELL", "kafka-topics.sh --bootstrap-server 127.0.0.1:9092 --topic to-websockets-candles --describe"]
+      interval: 5s
+      timeout: 20s
+      retries: 50
+  postgres:
+    build:
+      context: ../indexer
+      dockerfile: ../indexer/Dockerfile.postgres.local
+    ports:
+      - 5435:5432
+    environment:
+      POSTGRES_PASSWORD: dydxserver123
+      POSTGRES_USER: dydx_dev
+      DATADOG_POSTGRES_PASSWORD: dydxserver123
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U dydx_dev"]
+      interval: 5s
+      timeout: 20s
+      retries: 10
+  redis:
+    image: redis:5.0.6-alpine
+    ports:
+      - 6382:6379
+  dydxprotocold0:
+    image: local:e2etest-dydxprotocol
+    entrypoint:
+      - cosmovisor
+      - run
+      - start
+      - --log_level
+      # Note that only this validator has a log-level of `info`; other validators use `error` by default.
+      # Change to `debug` for more verbose log-level.
+      - info 
+      - --home
+      - /dydxprotocol/chain/.alice
+      - --p2p.persistent_peers 
+      - "17e5e45691f0d01449c84fd4ae87279578cdd7ec@dydxprotocold0:26656,47539956aaa8e624e0f1d926040e54908ad0eb44@dydxprotocold2:26656,5882428984d83b03d0c907c1f0af343534987052@dydxprotocold3:26656"
+      - --bridge-daemon-eth-rpc-endpoint
+      - "${ETH_RPC_ENDPOINT}"
+    environment:
+      - DAEMON_HOME=/dydxprotocol/chain/.alice
+    volumes:
+      - ../protocol/localnet/dydxprotocol0:/dydxprotocol/chain/.alice/data
+    ports:
+      - "26657:26657"
+      - "9090:9090"
+      - "1317:1317"
+  
+  dydxprotocold1:
+    image: local:e2etest-dydxprotocol
+    entrypoint:
+       - cosmovisor
+       - run
+       - start
+       - --log_level
+       - error
+       - --home
+       - /dydxprotocol/chain/.bob
+       - --p2p.persistent_peers 
+       - "17e5e45691f0d01449c84fd4ae87279578cdd7ec@dydxprotocold0:26656,b69182310be02559483e42c77b7b104352713166@dydxprotocold1:26656,47539956aaa8e624e0f1d926040e54908ad0eb44@dydxprotocold2:26656,5882428984d83b03d0c907c1f0af343534987052@dydxprotocold3:26656"
+       - --non-validating-full-node=true
```

Comment:
nit: Does this work? In your `local.sh` `bob` is given some stake and is a validator, but the node here is set to be a full-node? Perhaps it's better to create a separate full-node for the indexer? Non-blocking if this works for now, but maybe a comment on how this works.